crates/rerun_c/src/lib.rs

```diff
@@ -534,8 +540,11 @@ fn parse_arrow_ipc_encapsulated_message(
     let chunks = chunks.map_err(|err| format!("Arrow error: {err}"))?;
 
     if chunks.is_empty() {
-        return Err("No Chunk found in stream".to_owned());
+        // This might be a bug in arrow2 or the C++ SDK, but it seems that when we send an
+        // IPC with an empty array from C++ we don't get any arrays here at all.
+        return Ok(arrow2::array::NullArray::new_empty(arrow2::datatypes::DataType::Null).boxed());
```

Comment:
That's going to throw an error at runtime, we cannot blindly change the type of whatever was being logged to `Null`.

Hitting this path should definitely return an error; I'm looking into the root cause.